components/script/dom/baseaudiocontext.rs

```diff
@@ -225,7 +225,7 @@ impl BaseAudioContext {
                             if this.state.get() != AudioContextState::Running {
                                 this.state.set(AudioContextState::Running);
                                 let window = DomRoot::downcast::<Window>(this.global()).unwrap();
-                                window.dom_manipulation_task_source().queue_simple_event(
+                                window.task_manager().dom_manipulation_task_source().queue_simple_event(
```

Comment:
It's also borrowed before.

To be fair, this parts also bothered me. I suspect that there's some way in which this code can be refactor to call less `clone`s. But I'm not sure it's immediately obvious.

Maybe I can offer you to open an issue to track the tech debt?

Reply:
Ok, thanks for the info. I'm not sure at this point how to make a potential issue actionable, so I will not open one. If you happen to have a clearer idea, it could be useful to open one indeed...

Reply:
No, not really.